tests/testthat/test-vendor.R

```diff
@@ -24,8 +24,8 @@ describe("cpp_vendor", {
 
     cpp_vendor(pkg_path(pkg))
```

Comment:
```suggestion
    cpp_vendor(file.path(pkg_path(pkg), "inst/include"))
```

Also add a test for vendoring into another directory?